[2025-02-23T15:00:03.810+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:00:03.856+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: newyork_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [queued]>
[2025-02-23T15:00:03.872+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: newyork_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [queued]>
[2025-02-23T15:00:03.872+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-23T15:00:03.897+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-23 14:00:00+00:00
[2025-02-23T15:00:03.905+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=20273) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-23T15:00:03.906+0000] {standard_task_runner.py:72} INFO - Started process 20278 to run task
[2025-02-23T15:00:03.906+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'newyork_weather_pipeline', 'transform_weather_data', 'scheduled__2025-02-23T14:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/newyork_weather_pipeline/newyork_weather_etl.py', '--cfg-path', '/tmp/tmp14r_c8_t']
[2025-02-23T15:00:03.908+0000] {standard_task_runner.py:105} INFO - Job 140: Subtask transform_weather_data
[2025-02-23T15:00:03.998+0000] {task_command.py:467} INFO - Running <TaskInstance: newyork_weather_pipeline.transform_weather_data scheduled__2025-02-23T14:00:00+00:00 [running]> on host ee2081ae3a04
[2025-02-23T15:00:04.173+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='newyork_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T14:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-23T14:00:00+00:00'
[2025-02-23T15:00:04.174+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:00:04.178+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '40.7128', 'longitude': '74.0060', 'temperature': -10.6, 'windspeed': 9.0, 'winddirection': 88, 'weathercode': 1}
[2025-02-23T15:00:04.226+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:00:04.226+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=newyork_weather_pipeline, task_id=transform_weather_data, run_id=scheduled__2025-02-23T14:00:00+00:00, execution_date=20250223T140000, start_date=20250223T150003, end_date=20250223T150004
[2025-02-23T15:00:04.323+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:00:04.373+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:00:04.376+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
